cocotb_build_libs.py

```diff
@@ -343,7 +384,7 @@ def _get_common_lib_ext(include_dir, share_lib_dir):
     ]
     if os.name == "nt":
         libcocotbutils_sources += ["libcocotbutils.rc"]
-    libcocotbutils_libraries = ["gpilog"]
+    libcocotbutils_libraries = _prefix_libraries(["gpilog"])
```

Comment:
Why are we prefixing the libraries here? [this answer](https://stackoverflow.com/a/9796089/102441) seems to suggest the the library names should be unprefixed.

Reply:
If you pass `-lcocotb` to mingw it looks for `libcocotb.dll` while msvc looks for `cocotb.dll`. As we probably still want the library to be called `libcocotb.dll` we prefix it with lib if we compile with msvc.

Reply:
I don't really agree with that. Under msvc, "libpython" is really called `python.dll`. I think we should try to stick with the platform defaults unless there is a good reason not to.

Presumably the issue is that the makefiles assume this prefix?

Reply:
In this case name of vpi/fli/vhpi library will depend on the compiler one uses? So depending on compiler mings/msvc different name has to be used?

Reply:
Is not better to have a bit more "ugly" compilation and keep those names the same?